spec/person_spec.rb

```diff
@@ -0,0 +1,31 @@
+require './person'
+require './student'
+require './teacher'
+require './rental'
+require './book'
+
+describe Person do
+  context 'When testing the Person class' do
+    it 'Should Create a Person' do
+      person = Person.new(22, 'Jhon')
+      expect(person.name).to eq('Jhon')
+      expect(person.age).to eq(22)
+    end
+
+    it 'Should be instance of Person' do
+      person = Person.new(22, 'Jhon')
+      expect(person).to be_instance_of(Person)
+    end
+
+    it 'Should have a correct name' do
+      person = Person.new(22, 'Jhon')
+      expect(person.correct_name).to eq('Jhon')
+    end
+
+    it 'Should Create a Rental' do
+      book = Book.new('React', 'Jhon')
+      rental = Rental.new('2020-10-10', book, self)
+      expect(rental.book.title).to eq('React')
+    end
+  end
```

Comment:
- [ ] Good job with test coverage so far, but since we are aiming for 100% test coverage, it will be nice to cover all our methods in all our classes with tests, for example, for this `Person Class`, the following methods should be covered with tests: 
  - When checking permissions
  - When validating name
### Please ensure all your methods in all your classes are covered with tests, not excluding `all classes that you created while implementing the Decorator design pattern` as specified in the [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_unit_tests.md)